test/rqg/datasets/star_schema.sql

```diff
@@ -42,15 +42,19 @@ CREATE TABLE d1 (pk1 INT, pk2 INT NOT NULL, v INT NOT NULL);
 CREATE INDEX d1_i1 ON d1(pk1, pk2);
 
 INSERT INTO d1 VALUES
+  -- pk1 != pk2 rows
   (NULL, 0, 0),
   (0, 0, 1),
   -- 1 not present in d1
   (2, 3, 0), (3, 2, 1),
   (3, 4, 0),
   -- 4 has no rows in d1
-  (5, 6, 0),
-  (6, 7, 0)
-  -- 7 is not present in either table
+  (5, 6, 0), (6, 5, 1),
+  (6, 7, 0),
+
+  -- pk1 = pk2 rows
+  (3, 3, 0),
```

Comment:
Adding rows with identical `(pk1, pk2)` values to some `dx` tables to produce non-NULL results in left joins of the form

```
LEFT JOIN dim_table AS dx ON( ft.fk1 = dx.pk2 AND ft.fk1 = dx.pk1 )
```

or

```
LEFT JOIN dim_table AS dx ON( dy.pk1 = dx.pk2 AND dy.pk1 = dx.pk1 )
```